src/pages/en/guides/content-collections.md

```diff
@@ -0,0 +1,480 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Content Collections (Experimental)
+description: Content collections help organize your Markdown and type-check your frontmatter with schemas.
+i18nReady: false
+---
+
+Content collections help organize your Markdown or MDX and type-check your frontmatter with schemas. You may reach for collections if you:
+- **Have a medium-to-large number of documents** to manage and fetch (example: a blog with 50+ posts).
```

Comment:
`medium-to-large` and `50+ posts` in the same sentence feels weird. For a blog it kinda makes sense, as 50 posts is a fair amount for most blogs, but for users that have anything else that is smaller than blog posts, 50 posts is absurdly small

Reply:
Fair enough! Everyone has a different definition of "medium." How about "non-trivial number of posts?"